Ryujinx/Ui/MainWindow.cs

```diff
@@ -827,11 +830,13 @@ private void Row_Activated(object sender, RowActivatedArgs args)
         private void VSyncStatus_Clicked(object sender, ButtonReleaseEventArgs args)
         {
             _emulationContext.EnableDeviceVsync = !_emulationContext.EnableDeviceVsync;
+            Logger.Notice.Print(LogClass.Application, $"VSync changed to: {_emulationContext.EnableDeviceVsync}");
         }
 
         private void DockedMode_Clicked(object sender, ButtonReleaseEventArgs args)
         {
             ConfigurationState.Instance.System.EnableDockedMode.Value = !ConfigurationState.Instance.System.EnableDockedMode.Value;
+            Logger.Notice.Print(LogClass.Application, $"IsDocked changed to: {ConfigurationState.Instance.System.EnableDockedMode.Value}");
```

Comment:
```suggestion
            Logger.Notice.Print(LogClass.Application, $"IsDocked toggled to: {ConfigurationState.Instance.System.EnableDockedMode.Value}");
```
and add a new empty line before